presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentConfig.java

```diff
@@ -46,8 +45,13 @@ default String getConfigName()
         return nameForConfigClass(getClass());
     }
 
-    default Optional<EnvironmentExtender> extendEnvironment(String environmentName)
+    default EnvironmentExtender getEnvironmentExtender(String environmentName)
     {
-        return Optional.empty();
+        return builder -> builder;
+    }
+
+    default Environment.Builder applyToEnvironment(String environmentName, Environment.Builder builder)
```

Comment:
this name is misleading to me. Why not `extendEnvironment`?

Reply:
👍🏻